controllers/shopCreateController.js

```diff
@@ -0,0 +1,21 @@
+// shopsSongsController
```

Comment:
I don't think you use this file - there is no route to it? Maybe this should be deleted. Also again, if you're cutting and pasting from another project make sure you remove the comments (or change them) in the current project.